addOns/diff/src/main/javahelp/org/zaproxy/zap/extension/diff/resources/help/contents/diff.html

```diff
@@ -12,7 +12,9 @@ <H1>Diff</H1>
 	The Diff add-on allows you to compare 2 requests or responses.
 </p>
 <p>
-	It is typically accessed via right click menu items.
+	It can be accessed by selecting two requests from the site tree and then pressing the right-click button. You'll see the options "Compare 2 Requests", "Compare 2 Responses" there clicking one of it will open the diff window. 
```

Comment:
- It can be accessed from any view component that shows more than one message, not just the Sites tree.
- Better to use "context menu" than "right-click button" as it's not mouse dependent (one can use the keyboard to show the context menu).
- I'd be more generic and say "messages" not "requests" since it's for both requests and responses.